changelog.txt

```diff
@@ -1,3 +1,31 @@
+1.11.1 (2021-04-17)
+------------------------------------------------------------------------
+Feature: Toggle to enable/disable vsync (#8997)
+Feature: Volume controls in the Game Options window, and better defaults (#8943)
+Add: Better plural support for Romanian (#8936)
+Change: Improve layout and spacing of several windows at different GUI scales (#9041, #9042, #9044)
+Change: [Win32] Use user UI language setting for initial language selection (#8974)
+Change: Make effect volume scale more intuitively (#8945, #8950)
+Change: Improve padding of Object & Rail station windows (#8929)
+Fix #7513: [Script] Crash on garbage collection with misbehaving script (#9040)
+Fix #9028: [OpenGL] Crash when changing max sprite zoom level (#9032)
+Fix #8874: show a warning when a NewGRF scan is requested multiple times (#9022)
+Fix #9015: [Win32] Crash on running "pwd" command in the console (#9016)
+Fix #9008: Validate starting year given on the command line (-t) (#9014)
+Fix #8878: [Network] Slow DNS queries could block the server and disconnect clients (#9013)
+Fix: Improve validation of OpenGL video driver to avoid crashes (#9007)
+Fix: Credits scrolled too slowly with larger font sizes (#8994)
+Fix #8977: Crash when altering max sprite resolution (#8993)
+Fix #8956: Industry disaster news messages showed the wrong location (#8992)
+Fix: [Win32] Font glyphs of certain widths had broken rendering (#8990)
+Fix #8930: [Win32] Duplicate text input issue for systems using IME (#8976)
+Fix: [Network] Potential stale client entries in client list (#8959)
+Fix: Graphical issues when dragging measurement tooltips (#8951)
+Fix: [Fluidsynth] Use provided default soundfont if available (#8948, #8953)
+Fix #8935: [macOS] Crash on save (#8944)
+Fix #8922: Crash when selling shared vehicles with shared vehicle window open (#8926)
+Fix: Compiling on armhf (Raspberry Pi) (#8924)
+
```

Comment:
Before your own OCD triggers next round: missing one newlines :) (should be 2 instead of 1, looking at the rest)